extractor/restriction_parser.cpp

```diff
@@ -152,6 +153,21 @@ RestrictionParser::TryParse(const osmium::Relation &relation) const
             {
                 is_only_restriction = true;
             }
+            if (strlen(iter->key()) > 11)

```

Comment:
Yep, `11` because `restriction` is length 11, so if it's longer than 11 it's a [conditional restriction](http://wiki.openstreetmap.org/wiki/Conditional_restrictions#Transportation_mode) (`restriction:<transportation_mode>`)